packages/react/src/entries/main.ts

```diff
@@ -0,0 +1,29 @@
+import { datadogRum } from '@datadog/browser-rum'
+import type { RumInitConfiguration } from '@datadog/browser-rum'
+import { _computeViewName } from '../lib/_computeViewName'
+import type { RouteMatch, Router, RouterState } from './types'
+
+export const createDatadogReactRouter = <T extends Router>(
+  router: T,
+  computeViewName: (routeMatches: RouteMatch[]) => string = _computeViewName
+) => {
+  router.subscribe((state: RouterState) => {
+    const viewName = computeViewName(state.matches)
+    datadogRum.startView(viewName)
+  })
+  return router
+}
+
+export const datadogReactRum = Object.assign({}, datadogRum, {
```

Comment:
<!-- Thoughts represent an idea that popped up from reviewing. These comments are non-blocking by nature, but they are extremely valuable and can lead to more focused initiatives and mentoring opportunities.  -->
**💭 thought:** ‏I am wondering if we should not override the global variable DD_RUM in this case.
cf https://github.com/DataDog/browser-sdk/blob/7b87e74044cdd92b34383fd3ce48e953054f6597/packages/rum/src/entries/main.ts#L40